global_parameter_server:
  ros__parameters:
    robot_list: 
      - "Test2"